Common/Tests/Utilities.UI/UI/ComboBox.cs

```diff
@@ -14,27 +14,88 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
-using System.Collections.Generic;
-using System.Linq;
+using System;
 using System.Windows.Automation;
+using System.Windows.Input;
 
 namespace TestUtilities.UI {
     public class ComboBox : AutomationWrapper {
         public ComboBox(AutomationElement element)
             : base(element) {
         }
 
-        public void SelectItem(string name) {
+        public void LogAvailableItems()
+        {
             ExpandCollapsePattern pat = (ExpandCollapsePattern)Element.GetCurrentPattern(ExpandCollapsePattern.Pattern);
             pat.Expand();
-            try {
-                var item = Element.FindFirst(TreeScope.Descendants, new PropertyCondition(AutomationElement.NameProperty, name));
+            try
+            {
+                var items = Element.FindAll(TreeScope.Descendants, new PropertyCondition(AutomationElement.ControlTypeProperty, ControlType.ListItem));
+                Console.WriteLine("Available items in the ComboBox:");
+                foreach (AutomationElement item in items)
+                {
+                    Console.WriteLine($"- {item.Current.Name}");
 
-                if (item == null) {
+                }
+            }
+            finally
+            {
+                pat.Collapse();
+            }
+        }
+        public void SelectItem(string name)
+        {
+            ExpandCollapsePattern pat = (ExpandCollapsePattern)Element.GetCurrentPattern(ExpandCollapsePattern.Pattern);
+            pat.Expand();
+            try
+            {
+                AutomationElement itemFirst = null;
+                var items = Element.FindAll(TreeScope.Descendants, new PropertyCondition(AutomationElement.ControlTypeProperty, ControlType.ListItem));
+                Console.WriteLine("Available items in the ComboBox:");
+                foreach (AutomationElement item in items)
+                {
+                    Console.WriteLine("controls");
+                    Console.WriteLine($"{item.Current.Name}");
+                    if (item.Current.Name.Contains(name))
```

Comment:
use contains instead of equals because now the automation name is super long with env name and path.